libcxx/test/std/containers/views/views.span/span.elem/op_idx.pass.cpp

```diff
@@ -41,7 +41,6 @@ void testRuntimeSpan(Span sp, std::size_t idx)
     assert(r1 == r2);
 }
 
-struct A{};
```

Comment:
Unrelated change?

Reply:
Yes, I can restore that change. Initially I was looking into reusing op_idx tests and I noticed that this declaration is unused.